.circleci/config.yml

```diff
@@ -466,10 +460,9 @@ workflows:
         - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
         - equal: [ "load-shedder-integration-tests", << pipeline.schedule.name >> ]
     jobs:
-      - purchases-load-shedder-integration-tests-build
       - run-firebase-tests-purchases-load-shedder-integration-test:
-          requires:
-            - purchases-load-shedder-integration-tests-build
+          context:
+            - slack-secrets
```

Comment:
which channel is this sending to?

Reply:
#feed-sdk-load-shedder-integration-tests